numpyro/distributions/__init__.py

```diff
@@ -5,10 +5,15 @@
     BetaBinomial,
     DirichletMultinomial,
     GammaPoisson,
+    NegativeBinomial2,
+    NegativeBinomialLogits,
+    NegativeBinomialProbs,
```

Comment:
It would be nice to have `NegativeBinomial` which dispatches to either of those two classes. Similar to `Bernoulli` dispatching to `BernoulliLogits` and `BernoulliProbs`